build_tools/cmake/iree_bytecode_module.cmake

```diff
@@ -70,6 +70,15 @@ function(iree_bytecode_module)
 
   iree_get_executable_path(_COMPILE_TOOL_EXECUTABLE ${_COMPILE_TOOL})
 
+  if(CMAKE_SYSTEM_PROCESSOR STREQUAL "riscv" AND
+    RISCV_CPU STREQUAL "rv64" AND
```

Comment:
We talked about in https://github.com/iree-org/iree/pull/10330#discussion_r966167073 to consolidate all the cross-compile config somehow in a later PR.

To answer your question. rv64 is RISC-V + 64-bit + Linux OS. Most of our build targets are assumed to run on an OS, except for `rv32-baremetal`, so we don't have the `-linux` appended for the RISCV_CPU config
https://github.com/iree-org/iree/blob/main/build_tools/cmake/riscv.toolchain.cmake#L48

Reply:
Yeah more consistent "run configuration" would definitely be nice. My less involved suggestion here was that maybe instead of having a variable like `RISCV_CPU` that is `RISC-V` specific, we should be using a generic variable. Actually, looking into it a bit, I'm surprised that `CMAKE_SYSTEM_PROCESSOR` isn't already being set to something more specific, like riscv_64. It looks like it does get more specific on x86 with `x86_64`. https://stackoverflow.com/q/70475665 and here I see someone setting it to specifically rv32: https://github.com/five-embeddev/riscv-scratchpad/blob/c3cf48324ea041251f4452d2a16449d9bd344370/cmake/cmake/riscv.cmake#L50. Anyway, this is not for this PR, just something that popped out at me